### DISCUSSION TITLE
Colors on the CLI

### DISCUSSION THREAD

**Discussion by litchipi** [massalabs/massa#3671] [Show and tell]
While implementing https://github.com/massalabs/massa/issues/3294, I had to make choices on what colors to use to display important informations.

I organised styles by their utility in a `Style` enum with a variant for each type of information, and the method `style` to generate a styled text, to be used like this: `Style:Id.style(some_id)`

Here is the variants and their colors that I got now:

![Capture d’écran du 2023-03-09 13-59-07](https://user-images.githubusercontent.com/61109829/224030386-c0e69a73-091f-441f-a34d-efa539f9688f.png)

You can reproduce this by:
- Check out `test_cli_colors` branch
- In `massa/massa-client`, run `cargo run`
- In the client, write the command `wallet_info`

Some other examples:
![Capture d’écran du 2023-03-09 14-04-57](https://user-images.githubusercontent.com/61109829/224032205-aadd940f-a3e1-470b-ac5c-77aaa5391708.png)
![Capture d’écran du 2023-03-09 14-02-21](https://user-images.githubusercontent.com/61109829/224032190-aaa800ad-aaa0-42da-b974-bf7452213111.png)
![Capture d’écran du 2023-03-09 14-05-58](https://user-images.githubusercontent.com/61109829/224032226-0e98d279-d96a-4843-9349-28c75e49e4f7.png)
![Capture d’écran du 2023-03-09 14-03-53](https://user-images.githubusercontent.com/61109829/224032200-e5fc1456-1a0b-4f34-87c1-b418aff37cb8.png)

**Comment by bilboquet** [comment]
As a matter of taste: overall, I like the selected colors

![image](https://user-images.githubusercontent.com/8612342/224084664-0dc396a6-295d-4f61-a260-571990765fe2.png)
Having 3 different colors for active, final and candidate here is a bit too much for me and it does not improve the readability.
Maybe here, I wish I see some red if, for my staking address,  active rolls == final rolls == candidate rolls == 0 (i.e. there is something wrong).

**Comment by modship** [comment]
It's like a cyberpunk theme
